website/static/js/folderpicker.js

```diff
@@ -52,11 +52,18 @@ function treebeardTitleColumn(item, col) {
             tb.updateFolder(null, item);
         };
     }
-
-    return m('span', {
-        className: cls,
-        onclick: onclick
-    }, item.data.name);
+    if (item.data.addon == "googledrive"){
+        return m('span', {
+            className: cls,
+            onclick: onclick
+        }, decodeURIComponent(item.data.name));
+    }
+    else {
+        return m('span', {
+            className: cls,
+            onclick: onclick
+        }, item.data.name);
```

Comment:
I think we can get rid of this conditional here and make this more concise by just calling `decodeURIComponent(item.data.name)` regardless of the addon.